src/_pytest/config/argparsing.py

```diff
@@ -27,6 +27,14 @@
 FILE_OR_DIR = "file_or_dir"
 
 
+class Notset:
+    def __repr__(self) -> str:
+        return "<notset>"
+
+
+notset = Notset()
```

Comment:
Minor, to keep the code standard for globals.

```suggestion
class NotSet:
    def __repr__(self) -> str:
        return "<notset>"


NOT_SET = NotSet()
```

Reply:
Thank you Bruno,
I like your suggestions. Will apply the changes and update the PR.

Regards